.circleci/config.yml

```
@@ -356,6 +364,7 @@ workflows:
   default-workflow:
     jobs:
       - build
+      - build-docker-image
```

we only need this for the release jobs (?) so I guess:
```suggestion
      - build-docker-image:
          filters:
            branches:
              ignore: /.*/
```

---

no, that's the contrary in fact.

---

oh, right